pkg/pool-weighted/test/ManagedPoolSettings.test.ts

```diff
@@ -951,15 +969,12 @@ describe('ManagedPoolSettings', function () {
               weightComplement: actualWeightComplement,
               lowerBound: actualLowerBound,
               upperBound: actualUpperBound,
-            } = await pool.getCircuitBreakerState(poolTokens.first);
-            const expectedWeightComplement = FP_ONE.sub(poolWeights[0]);
+            } = await pool.getCircuitBreakerState(poolTokens.get(tokenIndex));
+            const expectedWeightComplement = FP_ONE.sub(poolWeights[tokenIndex]);
             const totalSupply = await pool.getActualSupply();
-            const scalingFactors = await pool.getScalingFactors();
 
-            const expectedBptPrice = fpDiv(
-              fpMul(totalSupply, poolWeights[0]),
-              fpMul(initialBalances[0], scalingFactors[0])
-            );
+            // Don't scale; the getter will return the unscaled value.
+            const expectedBptPrice = fpDiv(fpMul(totalSupply, poolWeights[tokenIndex]), initialBalances[tokenIndex]);
```

Comment:
Shouldn't this test that the pool returns the bpt price that we set?

Reply:
I guess the computation here returns the same thing. We could replace it with `getUnscaledBptPrice()` though